src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -538,16 +538,12 @@ def gen_code(self, parent):
         zero_reduction_variables(calls, parent)
 
         clauses_str = self.default_clause._clause_string
```

Comment:
Please could you disable the pylint warning here or is there a getter we can use instead?